cdap-integration-test/src/test/java/co/cask/cdap/test/IntegrationTestBaseTest.java

```diff
@@ -40,4 +49,26 @@ public void testFlowManager() throws Exception {
     flowManager.stop();
   }
 
+  @Test
+  public void testDeployApplicationInNamespace() throws Exception {
+    Id.Namespace namespace = new Id.Namespace ("Test1");
+    createNamespace(new NamespaceMeta.Builder().setName(namespace).build());
+    ClientConfig clientConfig = new ClientConfig.Builder(getClientConfig()).build();
+    clientConfig.setNamespace(namespace);
+    ApplicationManager applicationManager = deployApplication(namespace, TestApplication.class);
+    FlowManager flowManager = applicationManager.getFlowManager(TestFlow.NAME).start();
+    flowManager.stop();
+    ApplicationClient applicationClient = new ApplicationClient(clientConfig);
+    Assert.assertEquals("TestApplication", applicationClient.list().get(0).getName());
+    applicationClient.delete("TestApplication");
+    Assert.assertEquals(0, new ApplicationClient(clientConfig).list().size());
+
+    // Check the default namespaces applications to see whether the application wasnt made in the default namespace
+    ClientConfig defaultClientConfig = new ClientConfig.Builder(getClientConfig()).build();
+    defaultClientConfig.setNamespace(Constants.DEFAULT_NAMESPACE_ID);
```

Comment:
you don't need this line, as getClientConfig uses ``default`` namespace by default.